modules/ding_content_rss/ding_content_rss.views_default.inc

```diff
@@ -0,0 +1,768 @@
+<?php
```

Comment:
Is the views export done with the newest DDB CMS core version as features have been upgraded in core. Just to prevent the view from being marked as overridden all the time :-)

Reply:
Not sure. What is the newest version? Can you refer to a ref/sha?

Reply:
DDB CMS master branch is always the last release :-)